model/form/TestCenterForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+namespace oat\taoOffline\model\form;
+
+use core_kernel_classes_Class;
+use core_kernel_classes_Resource;
+use oat\taoOffline\model\service\TaoOfflineTestCenterFormService;
+use oat\taoSync\model\synchronizer\custom\byOrganisationId\testcenter\TestCenterByOrganisationId;
+use tao_actions_form_Instance;
+use tao_helpers_form_FormFactory;
+use tao_helpers_Uri;
+
+class TestCenterForm extends tao_actions_form_Instance
+{
+    /**
+     * @var TaoOfflineTestCenterFormService
+     */
+    private $testCenterFormService;
+
+    /**
+     * @param TaoOfflineTestCenterFormService $testCenterFormService
+     * @param core_kernel_classes_Class $clazz
+     * @param core_kernel_classes_Resource|null $instance
+     * @param array $options
+     */
+    public function __construct (
+        TaoOfflineTestCenterFormService $testCenterFormService,
```

Comment:
I guess that you have to use ServiceManager to get this service (it could be configured)
so I guess you can create something like that:
```
private function getTestCenterFormService()
{
    return ServiceManager::instance()->get(TestCenterFormService::SERVICE_ID);
}
```